sherpa/astro/models/__init__.py

```diff
@@ -661,19 +661,19 @@ class Voigt1D(RegriddableModel1D):
 
         fwhm_l = fwhm_g / sqrt(2 * log(2))
 
-    An approximation for the FWHM of the profile, taken from [2]_,
+    An approximation for the FWHM of the profile, taken from [wiki_voigt]_,
     is
 
         0.5346 fwhm_l + sqrt(0.2166 fwhm_l^2 + fwhm_g^2)
 
     References
     ----------
 
-    .. [1] http://publikationen.badw.de/de/003395768
+    .. [Voigt_1912] http://publikationen.badw.de/de/003395768
```

Comment:
This website is now being served via a https server
```suggestion
    .. [Voigt_1912] https://publikationen.badw.de/de/003395768
```